.github/workflows/mdbook.yml

```diff
@@ -74,6 +72,23 @@ jobs:
         with:
           source-dir: ./book/
 
+      - name: Remove preview
+        if: github.event_name == 'pull_request' && github.event.action == 'closed'
+        run: |
+          PR_NUM=${{ github.event.number }}
+          CURRENT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+
+          git fetch origin gh-pages:gh-pages
+          git checkout gh-pages
+
+          rm -rf pr-preview/pr-$PR_NUM
+
+          git add -A
```

Comment:
can this be `git rm ...`?

Reply:
Fixed in https://github.com/ratatui-org/ratatui-book/pull/18/commits/ab6d54f3396f0d7b475a3b74be57c6cdcc4e5f36